impedance/models/circuits/elements.py

```diff
@@ -268,6 +268,33 @@ def Gs(p, f):
     return Z
 
 
+@element_metadata(num_params=4, units=['Ohm', '', '', '', ''])
+def Tas(p, f):
+    """ defines a Ls-de Levie Pore - Finite Length
+
+    Notes
+    ---------
+    .. math::
+        Z = \\frac{Z_0}{\\sqrt{gamma}\\tanh{\\sart{gamma}}}
```

Comment:
It looks like this doc string could benefit from some clean up. A few things I notice:

- First line Ls should be something like "defines a Finite-Length de Levie Pore element as represented in [1]" to mirror the Gerisher and other more complex elements
- The math seems to have some formatting problems. You should be able to build the documentation locally by: `cd docs`, `make html`, open build/html/index.html
- I can't figure out how to push to your branch, but here's what I was playing around with

```
Notes
    ---------
    .. math::
        Z = \\frac{R_{\\Omega,p}}{\\sqrt{\\Lambda}\\tanh{\\sqrt{\\Lambda}}}

    where

    .. math::
        R_{\\Omega,p} = \\frac{\\rho l}{\\pi r^2},

    .. math::
        \\Lambda = \\frac{1}{A} + B (j \\omega)^\\phi,

    and

    .. math::
        A = \\frac{R_{ct}r}{2\\rho l^2} \\quad B = \\frac{2\\rho l^2 C_{dl}}{r}

    where :math:`R_{\\Omega,p}` = p[0] (Ohms) and
    :math:`A` = p[1], `B` = p[2], `\\phi` = p[3]

    [1] Lasia, A. Impedance of porous electrodes.
    Journal of Electroanalytical Chemistry 397, 27–33 (1995).
    `doi:10.1016/0022-0728(95)04177-5
    <https://doi.org/10.1016/0022-0728(95)04177-5>`_.
    """
```